TileStache/Goodies/VecTiles/transform.py

```diff
@@ -685,6 +685,32 @@ def overlap(feature_layers, base_layer, cutting_layer,
         target_attribute, cutting_attrs, keep_geom_type)
 
 
+# map from old or deprecated kind value to the value that we want
+# it to be.
+_deprecated_landuse_kinds = {
+    'station': 'substation',
```

Comment:
Is this going to cause us problems? There are a significant number of not-unique values between OSM keys, do we need to de the mapping based on the original (full) OSM key:value pair instead? 

For instance, will a public transit station be remapped to 'substation" with this code?